src/LibraryManager.Vsix/UI/Models/InstallDialogViewModel.cs

```diff
@@ -393,46 +442,55 @@ private static bool IsAnyFileSelected(IReadOnlyList<PackageItem> children)
                     DestinationPath = InstallationFolder.DestinationFolder,
                 };
 
-                // When "Include all files" option is checked, we don't want to write out the files to libman.json.
-                // We will only list the files when user chose to install specific files.
-                if (LibraryFilesToInstall == FileSelectionType.ChooseSpecificFilesToInstall)
+                if (IsLibraryInstallationStateValid())
```

Comment:
>if (IsLibraryInstallationStateValid()) [](start = 16, length = 38)

Could this be moved to the beginning of the method?

Reply:
Updated in iteration 7

---
In reply to: [201160608](https://github.com/aspnet/LibraryManager/pull/293#discussion_r201160608) [](ancestors = 201160608)